indexer/indexer_tests/osm_editor_test.cpp

```diff
@@ -754,6 +764,8 @@ void EditorTest::IsCreatedFeatureTest()
   auto const mwmId = ConstructTestMwm([](TestMwmBuilder & builder)
   {
     builder.Add(TestCafe(m2::PointD(1.0, 1.0), "London Cafe", "en"));
+
+    builder.Add(TestPOI(m2::PointD(10, 10), "Corner Post", "default"));
```

Comment:
Another way is to add a parameter to ConstructTestMwm, like m2::RectD const mwmBounds. And it will be now a task for TestMwmBuilder to guarantee that mwm won't be smaller than mwmBounds.